content/applications/general/users/access_rights.rst

```diff
@@ -2,66 +2,148 @@
+Administration includes either :guilabel:`Settings` or :guilabel:`Access Rights`.
 
-.. image:: access_rights/groups-menus.png
+.. image:: access_rights/user-permissions-dropdown-menu.png
    :align: center
-   :height: 330
-   :alt: View of a group’s form emphasizing the tab menus in Odoo
+   :alt: Open the drop-down menu next to each app to set the user's level of permissions.
 
-*Access Rights* rules are the first level of rights. The field is composed of the object name, which
-is the technical name given to a model. For each model, enable the following options as appropriate:
+Creating and modifying groups
+=============================
 
-- *Read*: the values of that object can be only seen by the user.
-- *Write*: the values of that object can be edited by the user.
-- *Create*: values for that object can be created by the user.
-- *Delete*: the values of that object can be deleted by the user.
+*Groups* are app-specific sets of permissions that are used to manage common access rights for a
+large amount of users. Administrators can modify the existing groups in Odoo or create new ones to
+define rules for models within an application.
 
-.. image:: access_rights/groups-access-rights.png
+To access groups, first activate Odoo's :ref:`developer mode <developer-mode>`, then go to
+:menuselection:`Settings --> Users & Companies --> Groups`.
+
+.. image:: access_rights/click-users-and-companies.png
    :align: center
-   :alt: View of a group’s form emphasizing the tab access rights in Odoo
+   :alt: In developer mode, go to the settings page and click Users and Companies, then Groups.
+
+To create a new group, click :guilabel:`New`, select an application, and complete the New Group form
+(detailed below). To modify existing groups, click on a group and edit the contents of the form.
 
-| As a second layer of editing and visibility rules, *Record Rules* can be formed. They overwrite,
-  or refine, the *Access Rights*.
-| A record rule is written using a *Domain*. Domains are conditions used to filter or searching
-  data. Therefore, a domain expression is a list of conditions. For each rule, choose among the
-  following options: *Read*, *Write*, *Create* and *Delete* values.
+.. important::
+   Always test the settings being changed to ensure they are being applied to the correct users.
 
-.. image:: access_rights/groups-record-rules.png
+The group form contains multiple tabs for managing all elements of the group. In each tab, click
+:guilabel:`Add a line` to add a new row for users or rules, and click the :guilabel:`X` icon to
+remove a row.
+
+.. image:: access_rights/groups-form.png
    :align: center
-   :alt: View of a group’s form emphasizing the tab record rules in Odoo
+   :alt: Click each tab in the groups form to modify the settings of the group.
+
+- :guilabel:`Users` tab: lists the current users in the group, users listed in black have
+  administrative rights. Users without administrative access appear in blue.
```

Comment:
```suggestion
- :guilabel:`Users` tab: lists the current users in the group. Users listed in black have
  administrative rights. Users without administrative access appear in blue.
```

Reply:
I know you allude to this in a generic way above, but it could be nice to quickly add that you can add users by clicking 'add a line' and so on... just to round everything out